build_tools/circle/build_doc.sh

```diff
@@ -183,6 +183,11 @@ ccache -s
 
 export OMP_NUM_THREADS=1
 
+if [[ "$CIRCLE_BRANCH" =~ ^main$ || -n "$CI_PULL_REQUEST" ]]
+then
```

Comment:
why do we need it in main ?

Reply:
Such that we have an changelog published on the `/dev` website as well. I thought that it could be handy.

Reply:
ah nice 👍